content/source/docs/cloud/api/invoices.html.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: "cloud"
+page_title: "Invoices - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+[JSON API document]: /docs/cloud/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+# Invoices API
+
+-> **Note:** The invoices API is only available in Terraform Cloud.
+
+Organizations on credit-card-billed plans may view their previous and upcoming invoices.
+
+## List Invoices
+
+This endpoint lists the previous invoices for an organization.
```

Comment:
Is this endpoint paginated? (I'm guessing the invoices pile up, after a while... or at least we'd hope so 😅)

If so let's add the pagination boilerplate, should be able to just grab it from one of the other pages that uses it.

Reply:
Good catch! This endpoint is indeed paginated, but it uses a weird bespoke pagination scheme for reasons that boil down to "Stripe". Added.